### PR TITLE
chore: remove hardcoded Sentry DSN

### PR DESCRIPTION
**Pull request by VinneyJ**
Removed hardcoded Sentry DSN and set the env var access.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -117,7 +118,7 @@ function reallyInitializeApp(routes) {
 export default function initializeApp(routes) {
   // set up logging when you're in production mode
   if (isProdMode()) {
-    Raven.config('https://e19420a2c46a4f97942553dfe8322cc4@sentry.io/1229723', {
+    Raven.config(process.env.SENTRY_DSN, {
```
Comment (kilemensi) [inline src/index.js:121]: Did you confirm that the env vars are passed to/available in JS? I'm assuming it's python that loads the .env?

Comment (VinneyJ) [reply src/index.js:121]: Yes, I confirmed. I exported the envs and wrote a small javascript script to check if the var was accessible, and it worked. Yes, Python loads the envs.

**Review by kilemensi — APPROVED** [review]

File: src/index.js (hunk 1 above)
Comment (thepsalmist) [inline src/index.js:121]: Not a blocker @VinneyJ but isn't it better if we define the sentry DSN as a const, rather than passing it here directly

Comment (VinneyJ) [reply src/index.js:121]: @thepsalmist sure I can do that.

Comment (VinneyJ) [reply src/index.js]: It's ready for review and merging. @kilemensi

**Review by thepsalmist — APPROVED** [review]

**Pull request merged**